src/components/FollowDisplay/subComponent/FollowElement.jsx

```diff
@@ -5,13 +5,13 @@ import FollowingButton from '../../followingButton/FollowingButton';
 import styled from 'styled-components';
 import variables from '../../../styles/variables';
 
-const FollowElement = ({ name }) => {
+const FollowElement = ({ name, width }) => {
```

Comment:
```suggestion
const FollowElement = ({ name="유저이름", width=40 }) => {
```
으로 수정해서 props의 초깃값을 지정할 수 있습니다 
초깃값을 지정하면 변수를 사용하는 부분에서 조건부렌더링이 필요없겠습니다!